Fix capitalization and punctuation (#1506)
grafana/tempo: docs/tempo/website/configuration/compression.md

---
title: Compression and encoding
weight: 5
---

# Compression and encoding

> Tempo can compress traces that it pushes into the backend. This requires a bit extra
> memory and CPU but reduces the amount of stored data.  Anecdotal tests suggest that `zstd` will

Reviewer: I feel this wording is unpolished.  Here's a suggestion.  Not a necessary change.
```suggestion
Tempo can compress traces that it pushes to backend storage. This requires extra
memory and CPU, but it reduces the quantity of stored data.  Anecdotal tests suggest that `zstd` will
```
